content/troubleshooting/common-issues.md

```diff
@@ -123,3 +123,28 @@ In such cases, you will often see an error message similar to this one:
 **Solution**:
 * Confirm that you have set the correct bundle identifiers for all targets in Xcode.
 * Verify that the bundle identifier set in Xcode matches the bundle identifier set in the Flutter workflow editor under code signing.
+
+
+### Common Podfile issue
```

Comment:
The title should be more descriptive about the issue, eg. iOS deployment target issue